src/js/components/DataChart/doc.js

```diff
@@ -0,0 +1,168 @@
+    gap: PropTypes.oneOfType([
+      PropTypes.oneOf([
+        'none',
+        'xxsmall',
+        'xsmall',
+        'small',
+        'medium',
+        'large',
+        'xlarge',
+      ]),
+      PropTypes.string,
+    ]).description(`The spacing between the axes and the Charts.`),
+    pad: padPropType.description(`Spacing around the outer edge of
+    the drawing coordinate area for bars and points to overflow into.`),
+    size: PropTypes.oneOfType([
+      PropTypes.oneOf(['fill']),
+      PropTypes.shape({
+        height: PropTypes.oneOfType([
+          PropTypes.oneOf([
+            'xxsmall',
+            'xsmall',
+            'small',
+            'medium',
+            'large',
+            'xlarge',
+            'fill',
+          ]),
+          PropTypes.string,
+        ]),
+        width: PropTypes.oneOfType([
+          PropTypes.oneOf([
+            'xxsmall',
+            'xsmall',
+            'small',
+            'medium',
+            'large',
+            'xlarge',
+            'fill',
+            'auto',
+          ]),
+          PropTypes.string,
+        ]),
+      }),
+    ])
+      .description(
+        `The size of the Charts. This does not include the axes
+      and any gap. It is passed through to the underlying Chart.`,
+      )
+      .defaultValue({ width: 'medium', height: 'small' }),
+    thickness: thicknessType.description(`Chart thickness given to all
+    Charts if not specified per Chart in 'chart'.`),
+    xAxis: PropTypes.oneOfType([
+      PropTypes.bool,
+      PropTypes.shape({
+        guide: PropTypes.bool,
+        key: PropTypes.string,
+        labels: PropTypes.number, // default undefined, all data points
+        render: PropTypes.func, // (dataIndex, axisIndex) => element
+      }),
+    ]).description(`x-axis configuration. 'guide' specifies that vertical
+    guide lines should be drawn under the Chart, one per label.
+    'key' specifies what property in the 'data' should be used as
+    any label content. 'labels' specifies how many labels to show.
+    'render' allows for custom rendering of the labels. It will be
+    called with the current data index and axis index and should return
+    the element to render: (dataIndex, axisIndex) => element.`),
+    yAxis: PropTypes.oneOfType([
+      PropTypes.bool,
+      PropTypes.shape({
+        guide: PropTypes.bool,
+        labels: PropTypes.number, // default 2, top and bottom
+        render: PropTypes.func, // (value, axisIndex) => element
+      }),
+    ]).description(`y-axis configuration. 'guide' specifies that horizontal
+    guide lines should be drawn under the Chart, one per label.
+    'labels' specifies how many labels to show.
+    'render' allows for custom rendering of the labels. It will be
+    called with the value and axis index and should return
+    the element to render: (value, axisIndex) => element`),
+  };
+
+  return DocumentedDataChart;
+};
+
+// export const themeDoc = {
+// };
```

Comment:
Agreed. I think this should just be a reference in the documentation, rather than duplicating the Chart theme documentation here. I'm also expecting to say that further theming capabilities are being explored, as part of this being experimental.